index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style1.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@1,500&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <header class="nav-container">
+        <div class="wrapper">
+            <img class="headerimgHolder" src="assets/image_geometry_1.png" alt="headerimgHolder" />
+            <div class="ic_menu">
+                <nav><img src="assets/ic_menu.png" alt="menu icon"></nav>
+            </div>
+        </div>
+
+    </header>
+
+    <section class="flex-container">
+        <h1 class="title_name">
+            Charlie
+            Giorgania
+        </h1>
+        <p class="about_description">
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+            works.
+            If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
+        <a><div class="collaborate_button">
+            <p id="start-collaboration" >start collaboration</p>
+        </div>
+        </a>
```

Comment:
- [ ] [OPTIONAL]: `start collaboration` is a `button` 🔘 , so it should be:
```
<button>
  start collaboration
</button>
```

Reply:
Yes, I will do that. I thought to use <a> which can be used to call function from js and have added on hover method in CSS to look like button